docs/overview/hooks.md

```diff
@@ -14,60 +14,84 @@ title: Lifecycle hooks overview
     <a style="font-style:italic;padding:5px;margin:5px;"  href="https://egghead.io/lessons/react-loading-data-from-the-server">Hosted on egghead.io</a>
 </details>
 
-All of the below hooks can be created by returning an action with the given name, like:
+`mobx-state-tree` supports passing a variety of hooks that are called throughout a node's lifecycle. Hooks are passes as actions with the name of the hook, like:
```

Comment:
**question:** @airhorns - why change this now? Was this just something you wanted to clean up, or do you feel like the changes in this PR specifically warrant the clarification?

I like these changes, but I'm just curious about the motivation for adding it to this PR.

Reply:
Was just trying to clarify it a bit while I was in here, can back this out if you prefer

Reply:
No, I love it. I was just curious. Thank you for taking the time to do so! Resolved.